core/templates/dev/head/pages/exploration_player/feedback_popup_directive.html

```diff
@@ -1,46 +1,3 @@
 <script type="text/ng-template" id="popover/feedback">
   <feedback-popup></feedback-popup>
 </script>
```

Comment:
Can we also remove this part?

Reply:
I tried using `popover-template=" 'path/to/feedback_popup_directive.html' "` instead of `popover-template=" 'popover/feedback' " but it's not working.

I also tried to directly put `<feedback-popup></feedback-popup>` in html as it's a single line and show it on `ng-if="show"` where show is set when someone clicks on the feedback icon.

It's working in the sense that the directive shows up. But behavior is not proper and it looks weird.

So, I couldn't get rid of this.